test/e2e/common/security_context.go

```diff
@@ -227,27 +227,27 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		}
 
 		/*
-		  Testname: allowPrivilegeEscalation unset and uid != 0.
-		  Description: Configuring the allowPrivilegeEscalation unset, allows the privilege escalation operation.
-		  A container is configured with allowPrivilegeEscalation not specified (nil) and a given uid which is not 0.
-		  When the container is run, the container is run using uid=0.
-		  This test is marked LinuxOnly since Windows does not support running as UID / GID, or privilege escalation.
+			Release : v1.14
+			Testname: Security Context, allowPrivilegeEscalation unset, uid != 0.
+			Description: Configuring the allowPrivilegeEscalation unset, allows the privilege escalation operation.
+			A container is configured with allowPrivilegeEscalation not specified (nil) and a given uid which is not 0.
+			When the container is run, container's output MUST match with expected output verifying container ran with uid=0.
 		*/
-		It("should allow privilege escalation when not explicitly set and uid != 0 [LinuxOnly] [NodeConformance]", func() {
+		framework.ConformanceIt("should allow privilege escalation when not explicitly set and uid != 0 [NodeConformance]", func() {
 			podName := "alpine-nnp-nil-" + string(uuid.NewUUID())
 			if err := createAndMatchOutput(podName, "Effective uid: 0", nil, 1000); err != nil {
 				framework.Failf("Match output for pod %q failed: %v", podName, err)
 			}
 		})
 
 		/*
-		  Testname: allowPrivilegeEscalation=false.
-		  Description: Configuring the allowPrivilegeEscalation to false, does not allow the privilege escalation operation.
-		  A container is configured with allowPrivilegeEscalation=false and a given uid (1000) which is not 0.
-		  When the container is run, the container is run using uid=1000.
-		  This test is marked LinuxOnly since Windows does not support running as UID / GID, or privilege escalation.
+			Release : v1.14
+			Testname: Security Context, allowPrivilegeEscalation=false.
+			Description: Configuring the allowPrivilegeEscalation to false, does not allow the privilege escalation operation.
+			A container is configured with allowPrivilegeEscalation=false and a given uid (1000) which is not 0.
+			When the container is run, container's output MUST match with expected output verifying container ran with given uid i.e. uid=1000.
 		*/
-		It("should not allow privilege escalation when false [LinuxOnly] [NodeConformance]", func() {
+		framework.ConformanceIt("should not allow privilege escalation when false [NodeConformance]", func() {
```

Comment:
This one is fine.

Reply:
Please do not remove the ``[LinuxOnly]`` label, test contains a SecurityContext, which is not working on Windows.

Reply:
Pushed new changes to promote this to Conformance.